includes/models/model.llms.quiz.php

```diff
@@ -218,7 +264,28 @@ public function can_be_resumed_by_student( $user_id = null ) {
 			$can_be_resumed_by_student = $last_attempt && $last_attempt->can_be_resumed();
 		}
 
-		return $can_be_resumed;
+		return $can_be_resumed_by_student;
+
+	}
+
+	/**
+	 * Gets quiz's last attempt key of a user.
+	 *
+	 * @since [version]
+	 *
+	 * @param int $user_id Optional. WP User ID, none supplied uses current user. Default `null`.
+	 * @return bool
+	 */
+	public function get_last_quiz_attempt_key( $user_id = null ) {
```

Comment:
I'd call it something like
`get_student_last_attempt_key()`
what do you think?

Reply:
yeah, sounds good. Updated.